libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -1339,6 +1422,20 @@ class AnswerWithJustification(BaseModel):
                 if is_pydantic_schema
                 else JsonOutputParser()
             )
+        elif method == "json_schema":
+            if schema is None:
+                raise ValueError(
+                    "schema must be specified when method is not 'json_mode'. "
+                    "Received None."
+                )
+            strict = strict if strict is not None else True
```

Comment:
to confirm: does `json_schema` with `strict=False` work?

Reply:
yep